tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -341,6 +341,45 @@ public void AddWebAppCallsProtectedWebApi_NoScopes()
             Assert.Contains(OidcConstants.ScopeProfile, oidcOptions.Scope);
         }
 
+        [Theory]
+        [InlineData(true, "http://localhost:123", "https://localhost:123")]
+        [InlineData(false, "http://localhost:123", "http://localhost:123")]
```

Comment:
Shouldn't there a case with an actual URL starting in https ?

Reply:
Updated.